plugin/pkg/auth/authenticator/token/bootstrap/bootstrap.go

```diff
@@ -49,6 +49,16 @@ type TokenAuthenticator struct {
 	lister internalversion.SecretNamespaceLister
 }
 
+// tokenErrorf prints a error message for a secret that has matched a bearer
+// token but fails to meet some other criteria.
+//
+//    tokenErrorf(secret, "has invalid value for key %s", key)
+//
+func tokenErrorf(s *api.Secret, format string, i ...interface{}) {
+	format = fmt.Sprintf("Bootstrap secret %s/%s matching bearer token ", s.Namespace, s.Name) + format
+	glog.V(3).Infof(format, i...)
```

Comment:
Don't know if this is the correct verbosity level. It was the existing one taken from `isSecretExpired`.